CHANGELOG.md

```diff
@@ -4,6 +4,602 @@ All notable changes to this project will be documented in this file.
 
 <!-- New entries go after this line -->
 
+## v3.0.0 - 2019-01-27
+
+For more details, [see the full changelog](https://github.com/thelounge/thelounge/compare/v2.7.1...v3.0.0) and [milestone](https://github.com/thelounge/thelounge/milestone/28?closed=1).
+
+Almost a year in the making, 1500+ commits, 650+ merged PRs, and 30+ contributors, The Lounge v3.0.0 really is the release of all the superlatives. It ships some of our most-upvoted and longest-awaited feature requests, with significant improvements all across the board, and a brand new documentation website.
+
+You will find a shortlist of the top changes below, but before we jump onto all the new and shiny things: if you are only looking for a simple list of **breaking changes**, we now have an [upgrade guide](https://thelounge.chat/docs/guides/upgrade). You can also try [our demo](https://demo.thelounge.chat) to see and test the new features.
```

Comment:
I'd add more emphasis to this, especially since it's a major release having it as a heading might help direct people who are like "tl;dr"

Reply:
Thanks for the feedback! Made the paragraph shorter, italics, and started it with "TL;DR".